apps/bare-expo/package.json

```
@@ -66,7 +66,7 @@
     "react-native-reanimated": "~3.10.0",
     "react-native-safe-area-context": "4.10.1",
     "react-native-screens": "~3.31.1",
-    "react-native-svg": "15.2.0-rc.0",
+    "react-native-svg": "^15.2.0",
```

We prefer to use strict versions or `~` in some cases to ensure we always have one version of the dependency installed in the monorepo.
```suggestion
    "react-native-svg": "15.2.0",
```